Documentation/scaling.rst

```diff
@@ -0,0 +1,88 @@
+Scaling
+=======
+
+When working with SI units in earthquake scenarios numbers might get very large.
+Rescaling the equations might be advantageous, e.g. when working with single precision arithmetic.
+In this section, we show how to properly scale the elastic wave equation.
+
+The elastic wave equation in velocity-stress form with source terms is given by
+
+.. math::
+
+   \begin{aligned}
+     \frac{\partial\sigma_{ij}}{\partial t}
+           - \lambda \delta_{ij}\frac{\partial u_k}{\partial x_k}
+           - \mu \left(
+               \frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i}
+            \right) &= s_i \\
+     \rho\frac{\partial u_i}{\partial t} - \frac{\partial \sigma_{ij}}{\partial x_j} &= f_i
+   \end{aligned}
+
+We define scaled quantities
+
+.. math::
+
+   \bar{x}_i = \frac{x_i}{L}, \quad \bar{u}_i = \frac{u_i}{u_c}, \quad
+       \bar{\sigma}_{ij} = \frac{\sigma_{ij}}{\sigma_c}
+
+Inserting these into the elastic wave equation gives
+
```

Comment:
A description of u_c, sigma_c is missing here. Probably good to add an example (e.g. change of unit). I would make this idea clear, because a standard approach would be non-dimensionalisation (e.g. in CFD)

Reply:
Also of L

Reply:
I called them "scaling constants" and added a link to the Example below which clarifies the concept.